<ui version="4.0" >
 <class>fm_TablePorts</class>
 <widget class="QWidget" name="fm_TablePorts" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="pb_up" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>icons:icon_up.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="pb_down" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>icons:icon_down.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="pb_insert" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>icons:icon_insert.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QPushButton" name="pb_remove" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset>icons:icon_minus.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="4" >
        <widget class="QComboBox" name="cb_insert" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTableView" name="tv_ports" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
